src/components/DatePicker/DatePicker.tsx

```diff
@@ -1,6 +1,9 @@
-import React, { useEffect, useState } from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import ReactDatePicker from 'react-datepicker'
 import 'react-datepicker/dist/react-datepicker.css'
+import { useSizeClass } from '../../hooks/useWindowSize'
```

Comment:
Try to use `useElementSize` which works on actual component instead of whole window. With `useElementSize` it will work better when we put few component in the same line.

Reply:
Hmm, I think it should works with window - we need to launch portal on mobile only so it should be not resize element (element has same width almost always). Will leave at it is or we can sync about that